sigs.yml

```
@@ -0,0 +1,502 @@
+- name: Specification SIGs
+  sigs:
+  - name: 'Specification: General'
+    meeting: Every Tuesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo
+    chat:
+      - type: slack
+        name: '#otel-specification'
+        id: C01N7PP1THC
+    invites: calendar-spec-general
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Specification: Logs'
+    meeting: 'Merged into "Specification: General"'
+    notes:
+      type: none
+      value: See above
+    chat:
+      - type: slack
+        name: '#otel-logs'
+        id: C01N5UCHTEH
+    invites: none
+    tcSponsors:
+    - name: Tigran Najaryan
+      github: tigrannajaryan
+    gcLiaison: []
+  - name: 'Specification: Sampling'
+    meeting: Every Thursday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1gASMhmxNt9qCa8czEMheGlUW2xpORiYoD7dBD7aNtbQ
+    chat:
+      - type: slack
+        name: '#otel-sampling'
+        id: C027DS6GZD3
+    invites: calendar-spec-sampling
+    tcSponsors:
+    - name: Josh MacDonald
+      github: jmacd
+    gcLiaison: []
+  - name: 'Specification: Configuration'
+    meeting: Every other Monday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 1uNgWQLQZcSVfBLRXfW9XCVJmKa5NH9R15fSOXhmpGWw
+    chat:
+      - type: slack
+        name: '#otel-config-file'
+        id: C0476L7UJT1
+    invites: calendar-spec-config
+    tcSponsors:
+    - name: Carlos Alberto
+      github: carlosalberto
+    - name: Jack Berg
+      github: jack-berg
+    gcLiaison: []
+  - name: 'Specification: Events'
+    meeting: Every Friday at 10:00 AM PT
+    notes:
+      type: gDoc
+      value: 1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A
+    chat:
+      - type: slack
+        name: '#otel-event'
+        id: C062HUREGUV
+    invites: calendar-spec-events
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Semantic Conventions: General'
+    meeting: Every Monday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8
+    chat:
+      - type: slack
+        name: '#otel-semantic-conventions-wg'
+        id: C041APFBYQP
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Armin Ruech
+      github: arminru
+    - name: Josh Suereth
+      github: jsuereth
+    - name: Reiley Yang
+      github: reyang
+    gcLiaison: []
+  - name: 'Semantic Conventions: Messaging'
+    meeting: Every Thursday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE
+    chat:
+      - type: slack
+        name: '#otel-messaging'
+        id: C02Q4AAHDSA
+    invites: calendar-semconv
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG